completions/removepkg

```diff
@@ -16,7 +16,7 @@ _removepkg()
 
     local root=${ROOT:-/}
     COMPREPLY=($(
-        cd "$root/var/log/packages" 2>/dev/null || return 1
+        command cd "$root/var/log/packages" 2>/dev/null || return 1
```

Comment:
```suggestion
        command cd -- "$root/var/log/packages" 2>/dev/null || return 1
```

I suppose `$ROOT` can contain something that would make this fail.

Reply:
Thank you for pointing this out! I somehow wrongly assumed that it will be treated as an ordinary argument when it contains / (just like `echo -e'a/b/c'`) but it was not the case. I also modified `completions/slackpkg`.

https://github.com/scop/bash-completion/blob/d8eab4776d1368e5a400cd244f543eaa437186d7/completions/slackpkg#L66

In `completions/slackpkg`, `$confdir` is currently a fixed string `/etc/slackpkg`

https://github.com/scop/bash-completion/blob/d8eab4776d1368e5a400cd244f543eaa437186d7/completions/slackpkg#L44

but it is not clear at a glance. Also, maybe some changes can be made in future, so I added `--` to the `cd` in `completions/slackpkg` as well. I pushed a new commit 1df7fde6e6e4eabe5a18e7f8b35dcd366173611f.